FWCore/Framework/test/DummyProxyProvider.h

```diff
@@ -4,7 +4,7 @@
 //
 // Package:     Framework
 // Class  :     DummyProvider
-// 
+//
```

Comment:
Were only cosmetic changes done to this file?

Reply:
Yes